docs/conf.py

```diff
@@ -0,0 +1,59 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+
+import os
+import sys
+
+import autoapi
+
+# Define path to the code to be documented **relative to where conf.py (this file) is kept**
+sys.path.insert(0, os.path.abspath('../src/'))
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'lsstseries'
+copyright = '2023, Doug Branton'
+author = 'Doug Branton'
+release = 'v0.1'
```

Comment:
[Aside] Should this use the project's version, or is the doc versioning distinct from library versioning?

Reply:
No, I don't think that the documentation versioning should be distinct from the code versioning. Good call here.

Reply:
I've added support for using `setuptools_scm` here in python-project-template with this PR: https://github.com/lincc-frameworks/python-project-template/pull/56